CMakeLists.txt

```diff
@@ -1,36 +1,42 @@
-cmake_minimum_required(VERSION 2.8)
-project(Kalman)
+cmake_minimum_required(VERSION 3.0)
+project(Kalman VERSION 0.1.0 LANGUAGES CXX)
+
+include(CTest)
+
+set(KALMAN_DEFINITIONS "-std=c++11")
```

Comment:
I wouldn't consider the used cpp standard a library definition, the application using the lib should decide which standard it wants to use (could be c++14 or later as well). Any specific reason why this is included here?